packages/playground/compile-wordpress/Dockerfile

```diff
@@ -152,18 +226,40 @@ RUN cd wordpress && \
     rm $phpfile.small; \
     done
 
-FROM emscripten as emscripten2
+
+FROM emscripten as emscripten3
 RUN rm -rf ./wordpress
-COPY --from=php /root/wordpress ./wordpress
+COPY --from=php2 /root /root/
 
 # === Postprocess WordPress ===
 
 # Disable load-scripts.php
-RUN cd wordpress && \
+RUN cd /root/wordpress && \
     sed "s/<?php/<?php define( 'CONCATENATE_SCRIPTS', false );/" wp-config.php > wp-config.php.new && \
     mv wp-config.php.new wp-config.php
 
-# Bundle the .data file using Emscripten 
+RUN ls /root/
+RUN if [[ -d wordpress-develop-trunk ]] ; then \
+    cp wordpress-develop-trunk/wp-tests-config-sample.php wordpress-develop-trunk/wp-tests-config.php && \
+    rm -rf wordpress-develop-trunk/build && \
+    mv wordpress wordpress-develop-trunk/build && \
+    mv wordpress-develop-trunk wordpress && \
+    mv /root/wp-cli.phar wordpress/wp-cli.phar && \
+    rm -rf wordpress/node_modules/ && \
+    rm -rf wordpress/src/js/_enqueues &&  \
+    rm -rf wordpress/vendor/{dealerdirect,doctrine,wp-coding-standards,phar-io,squizlabs} &&  \
+    rm -rf wordpress/test/{qunit,visual-regression,e2e,gutenberg} && \
```

Comment:
We removed a bunch of unneeded files to keep the file size small

Reply:
Awesome! Out of scope here, but later on it would be useful to actually run a smoke-test here to ensure the assumption about these dependencies holds over time.